src/main/java/com/mentoring/common/exceptions/ProductNotFoundException.java

```diff
@@ -0,0 +1,11 @@
+package com.mentoring.common.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+public class ProductNotFoundException extends ResponseStatusException {
+
+    public ProductNotFoundException() {
+        super(HttpStatus.NOT_FOUND, "Product not found.");
+    }
+}
```

Comment:
Another way of doing it, using annotations, is:

```
@ResponseStatus(HttpStatus.NOT_FOUND)
public class ProductNotFoundException extends RuntimeException {

    public ProductNotFoundException(final String message) {
        super(message);
    }
}
```

I think it's better to receive a message instead of hardcoding one here. This way you can write different messages for each time you thrown this exception.

Reply:
What about the package used to keep the exception. Would have you done differently?